clients.go

```diff
@@ -18,10 +18,10 @@ import (
 )
 
 type ClientsConfig struct {
-	// AddToScheme is an optional way to extend the known types to the gobal
+	Logger micrologger.Logger
+	// SchemeBuilder is an optional way to extend the known types to the global
 	// client-go scheme. Make use of it for custom CRs.
-	AddToScheme func(*runtime.Scheme) error
-	Logger      micrologger.Logger
+	SchemeBuilder []func(*runtime.Scheme) error
```

Comment:
I thought about providing custom type for that in this package.

E.g putting `type SchemeBuilder []func(*Scheme) error` in `types.go` and then replacing this with:

```suggestion
	SchemeBuilder SchemeBuilder
```


Then in the user code you can do:

```go
k8sclient.ClientsConfig{
	SchemeBuilder: k8sclient.SchemeBuilder{
		appv1alpha1.AddToScheme,
	},
}

Reply:
Nice and that is pretty aligned with the generated code.

@kopiczko @xh3b4sd PTAL